covid-vaccine-faq-bot/.env

```diff
@@ -0,0 +1,20 @@
+# description: Your Twilio phone number for sending and receiving messages
+# format: phone_number
+# required: true
+TWILIO_PHONE_NUMBER=
+
+# description: SID of the Vaccine Bot FAQ Studio Flow
+# configurable: false
+# format: sid
+FLOW_SID=
+
+# description: Your Google dialog flow project ID, which you can create at (https://cloud.google.com/dialogflow/cx/docs/quick/setup#project)  
+# format: text
+# required: true
+DIALOGFLOW_ES_PROJECT_ID=
```

Comment:
I'm wondering if this would be clearer if we name this `GOOGLE_CLOUD_PROJECT_ID`, because Dialogflow is really a sub-product inside GCP. It was confusing to me as I didn't know if it was an identifier that I needed within Dialogflow. Actually it's just the GCP Project ID.

Reply:
I agree, will change to GOOGLE_CLOUD_PROJECT_ID